source/index.txt

```diff
@@ -2,19 +2,129 @@
 :hidefeedback: header
 :noprevnext:
 
-.. default-domain:: mongodb
-
 =============================
 Start Developing with MongoDB
 =============================
 
-Connect your application to your database with one of our official libraries.
+Connect your application to your database with one of the official MongoDB 
+libraries.
+
+The following libraries are officially supported by MongoDB. MongoDB actively
+develops new features, adds performance enhancements, fixes bugs, and applies 
+security patches to them.
+
+.. card-group::
+   :columns: 3
+   :layout: default
+   :type: drivers
+
+   .. card::
+      :headline: C
+      :url: /c/
+      :icon: /figures/icons/c.svg
+      :icon-alt: C Driver icon
+
+   .. card::
+      :headline: C++
+      :url: /cxx/
+      :icon: /figures/icons/c++.svg
+      :icon-alt: C plus plus Driver icon
+
+   .. card::
+      :headline: C#
+      :url: /csharp/
+      :icon: /figures/icons/csharp.svg
+      :icon-alt: C sharp Driver icon
+
+   .. card::
+      :headline: Go
+      :url: https://www.mongodb.com/docs/drivers/go/current/
+      :icon: /figures/icons/go.svg
+      :icon-alt: Go Driver icon
+
+   .. card::
+      :headline: Java
+      :url: /java-drivers/
+      :icon: /figures/icons/java.svg
+      :icon-alt: Java Driver icon
+   
+   .. card::
+      :headline: Node.js
+      :url: https://www.mongodb.com/docs/drivers/node/current/
+      :icon: /figures/icons/node.svg
+      :icon-alt: Node Driver icon
+
+   .. card::
+      :headline: PHP
+      :url: /php/
+      :icon: /figures/icons/php.svg
+      :icon-alt: PHP Driver icon
+
+   .. card::
+      :headline: Python
+      :url: /python/
+      :icon: /figures/icons/python.svg
+      :icon-alt: Python Driver icon
+   
+   .. card::
+      :headline: Ruby
+      :url: https://www.mongodb.com/docs/ruby-driver/current/
+      :icon: /figures/icons/ruby.svg
+      :icon-alt: Ruby Driver icon
+
+   .. card::
+      :headline: Rust
+      :url: /rust/
+      :icon: /figures/icons/rust.svg
+      :icon-alt: Rust Driver icon
+
+   .. card::
+      :headline: Scala
+      :url: /scala/
+      :icon: /figures/icons/scala.svg
+      :icon-alt: Scala Driver icon
+
+   .. card::
+      :headline: Swift
+      :url: /swift/
+      :icon: /figures/icons/swift.svg
+      :icon-alt: Swift Driver icon
+
+   .. card::
+      :headline: TypeScript
+      :url: /typescript/
+      :icon: /figures/icons/ts.svg
+      :icon-alt: TypeScript Driver icon
+
+
+Featured Community Supported Libraries
```

Comment:
I: "Community-supported" is a [compound modifier](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#hyphens-in-compound-modifiers)

S:

```suggestion
Featured Community-Supported Libraries
```